app/main.py

```diff
@@ -1,3 +1,47 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import math
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as f:
+        data = json.load(f)
+        fuel_price = data["FUEL_PRICE"]
+        customers = data["customers"]
+        shops = [Shop(**shop) for shop in data["shops"]]
+    for people in customers:
+        customer = Customer(
+            people["name"],
+            people["product_cart"],
+            people["location"],
+            people["money"],
+            Car(people["car"]["brand"],
+                people["car"]["fuel_consumption"])
+        )
+        print(f"{customer.name} has {customer.money} dollars")
+        prices = []
```

Comment:
from task requirements:
When the customer arrives at the shop his location should equal to shop location. After the shop he arrives home and counts the remaining money.